app/src/pages/settings/AnnotationConfigDialog.tsx

```diff
@@ -0,0 +1,424 @@
+import React from "react";
+import { Controller, useFieldArray, useForm } from "react-hook-form";
+import { css } from "@emotion/react";
+
+import { Card } from "@arizeai/components";
+
+import {
+  Button,
+  Dialog,
+  FieldError,
+  Flex,
+  Form,
+  Icon,
+  Icons,
+  Input,
+  Label,
+  NumberField,
+  Radio,
+  RadioGroup,
+  TextArea,
+  TextField,
+  View,
+} from "@phoenix/components";
+import { useNotifyError, useNotifySuccess } from "@phoenix/contexts";
+import {
+  AnnotationConfig,
+  AnnotationConfigCategorical,
+  AnnotationConfigContinuous,
+  AnnotationConfigFreeform,
+  AnnotationConfigOptimizationDirection,
+  AnnotationConfigType,
+} from "@phoenix/pages/settings/types";
```

Comment:
```suggestion
} from "./types";
```
just to promote some level of directory encapsulation